### PR TITLE
Add latest tweets page

### PR DESCRIPTION
**Pull request by danielcti**
This PR adds a Latest Tweets page (issue #20)

TODO:
- [ ] Add a button in the homepage redirecting to this page
- [ ] Fetch more than 10 tweets, with pagination or infinite scroll.
- [ ] Add how long ago the tweet was added, i.e., (12min ago).

image

image

### CODE REVIEW

File: src/pages/latest_tweets.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { TweetData } from 'types/Tweet';
+import TweetInfo from 'components/TweetInfo';
+
+const LatestTweets: NextPage = () => {
+  const [data, setData] = useState<TweetData[]>([]);
+  const [isLoading, setLoading] = useState(false);
+
+  useEffect(() => {
```
Comment (sibelius) [inline src/pages/latest_tweets.tsx:13]: Why not serverSideProps?

File: src/pages/api/latest_tweets.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const MAX_RESULTS = 10;
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  const url =
+    `${BASE_URL}/${RECENT_TWEETS_URL}?query=${QUERY}&tweet.fields=${TWEET_FIELDS}` +
+    `&user.fields=${USER_FIELDS}&expansions=${EXPANSIONS}&max_results=${MAX_RESULTS}`;
+  const { data } = await axios.get<TwitterResponse>(url, {
```
Comment (sibelius) [inline src/pages/api/latest_tweets.ts:27]: Prefer to use pure fetch instead

**Review by sibelius — COMMENTED** [review]
Add readme.md information of how to get twitter token so it is easy for more people to contribute

File: src/pages/latest_tweets.tsx (hunk 1 above)
Comment (danielcti) [reply src/pages/latest_tweets.tsx:13]: Im not very used to nextjs, I will replace to serverSideProps.

**Comment by danielcti** [comment]
@sibelius do you think I need to implement the TODO tasks that I put on the PR description in this PR or we can do it later on?

**Comment by sibelius** [comment]
It can be in other pull requests

File: src/pages/api/latest_tweets.ts (hunk 2 above)
Comment (ferreira-mariana) [reply src/pages/api/latest_tweets.ts]: just out of curiosity, why?

**Pull request merged**